v2/emailpassword/common-customizations/signup-form/adding-fields.mdx

```diff
@@ -2,21 +2,35 @@
 id: adding-fields
 title: Adding Extra Fields
 hide_title: true
+show_ui_switcher: true
 ---
 
-import FrontendSDKTabs from "/src/components/tabs/FrontendSDKTabs"
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs";
 import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
 import TabItem from '@theme/TabItem';
 
 # Adding Extra Fields
 
+<PreBuiltOrCustomUISwitcher>
+
 ## Step 1: Front End
 
+<CustomUIContent>
+
+:::caution Not applicable
+You can skip this step in Custom UI, as you will be creating your own UI and already have control over the form fields. 
+:::
```

Comment:
instead of this, we should show them how to call the sign up function with extra form fields.

Reply:
See here (https://deploy-preview-751--admiring-bhabha-7b1be9.netlify.app/docs/thirdpartyemailpassword/custom-ui/email-password-login) for how we are telling users how to use our function for web, or how to call the API for mobile. So same thing needs to come here, but with extra form fields.